### ISSUE TITLE
Prediction from example code returns almost same values

### ISSUE THREAD

**Issue by iwatobipen** [recursionpharma/mole_public#1]
Dear developers,

First of all, thanks for sharing such as useful code for drug discovery!
I'm interested in MolE and I tested it with example code which is provided from README.md.
After running training code below
"mole_train model=finetune  data_file='data/TDC_Half_Life_Obach_train_seed0.parquet' checkpoint_path=null dropout=0.1 lr=1.0e-06 task=regression num_tasks=1 model.name='MolE_Finetune_Regression' model.hyperparameters.datamodule.validation_data='data/TDC_Half_Life_Obach_valid_seed0.parquet'"

I tested prediction but I got almost same predicted value from different molecules.
```python
from mole import mole_predict
import pandas as pd
df = pd.read_parquet('./data/TDC_Half_Life_Obach_valid_seed0.parquet')
smiles_list = df.smiles.to_list()[:60]
mole_predict.predict(smiles=smiles_list, 
                     task='regression',
                     num_tasks=1, 
                     pretrained_model="./lightning_checkpoint-epoch=4-step=70.ckpt", 
                     batch_size=32, num_workers=4)

array([[6.743252 ],
       [6.7186894],
       [6.7754207],
       [6.715815 ],
..snip
```
Should I train with more larger max_epochs (default is 100) to get more accurate model?
Any advice will be greatly appreciated.
Thanks!

@iwatobipen
